docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.md

```diff
@@ -131,19 +132,26 @@ class CustomerReader extends SprykerCustomerReader implements CustomerReaderInte
 }
 ```
 
-After applying the solution, re-evaluate the code. The same error shouldn’t be returned.
+After applying the solution, re-evaluate the code. The same error shouldn’t be returned. As soon as the extension point in core is released, refactor the code using the recommended [development strategies](/docs/scos/dev/back-end-development/extending-spryker/development-strategies/development-strategies.html).
+While it's not refactored, auto-upgrades are not supported, and the effort to update the project may be bigger and require more manual work.
 
+---
 
-## PrivateApi:Extension
+## PrivateApi:Extension: form class
 
-The following sections describe why this upgradability error can occur and how to solve it.
+It is not allowed to extend Private API code on the project level. Private API updates can break backward compatibility.
```

Comment:
Is there a more specific description for the form class, like for the facade?

Reply:
done